lib/github_changelog_generator/parser_file.rb

```diff
@@ -96,7 +96,9 @@ def convert_value(value, option_name)
       pr_wo_labels: :add_pr_wo_labels,
       pull_requests: :pulls,
       filter_by_milestone: :filter_issues_by_milestone,
-      github_api: :github_endpoint
+      github_api: :github_endpoint,
+      configure_sections: :configure_sections,
+      add_sections: :add_sections
```

Comment:
These are only needed here if they aren't the same - as you can see, all the other pairs are about irregular names.

So, do not add the new option names here.